src/App.js

```diff
@@ -2,8 +2,15 @@ import React, {useEffect, useState, useCallback} from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import styles from "styled-components";
+import "./index.css";
```

Comment:
Since we're using css modules, the proper import statement is the following:
```suggestion
import  style from "./App.module.css";
```
This will fix the issue of your styles not being applied to the `div` with the container classname.